src/components/Login/LoginForm/LoginCaption/LoginCaption.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './loginCaption.module.scss';
+import Caption from '../../../shared/Caption/Caption';
+
+const LoginCaption = () => (
+  <Caption classes={[styles.loginCaption]}>
+    Log in
+  </Caption>
+);
+
+export default LoginCaption;
```

Comment:
I think it's too big nesting in components
I't better to have single components on one level, not inside each other